deploy/operator/apply/03_MetacontrollerGatewaySynchHook.yaml

```diff
@@ -8,11 +8,7 @@ spec:
   parentResource:
     apiVersion: zalando.org/v1
     resource: "{{{ CRD_PLURAL }}}"
-  childResources:
-    - apiVersion: extensions/v1beta1
-      resource: ingresses
-      updateStrategy:
-        method: InPlace
+  childResources: []
```

Comment:
this isn't necessary here as we control that internally in our multi cluster deploy repo